Public/Src/IDE/VsCode/BuildXL.IDE.VsCode.dsc

```diff
@@ -89,23 +115,24 @@ namespace LanguageService.Server {
                             subfolder: a`projectManagement`,
                             contents: globR(d`client/projectManagement`)
                         },
+                        {
+                            subfolder: a`node_modules`,
+                            contents: [ Deployment.createDeployableOpaqueSubDirectory(VsCode.Client.installRootDir, r`node_modules`) ]
+                        },
+                        {
+                            subfolder: a`out`,
+                            contents: [ VsCode.Client.compileOutDir ]
+                        },
                         f`client/License.txt`,
                         f`client/package.nls.json`,
                         readme,
                         f`client/ThirdPartyNotices.txt`,
                         Branding.pngFile,
                         json,
-
-                        // This contains the actual extension source as well as the
-                        // node_modules that it depends on.
-                        Transformer.sealDirectory({
-                            root: d`pluginTemplate/extension`, 

```

Comment:
Note: I intentionally didn't delete the `pluginTemplate/extension` directory in this PR to avoid this PR having over 1000 modified files.  Once this PR gets merged, we should delete the `pluginTemplate/extension` folder in a separate PR (which won't need any thorough reviewing)